features/step_definitions/web_steps.rb

```diff
@@ -106,7 +106,7 @@ def with_scope(locator)
   end
 end
 
-When /^(?:|I )choose "([^"]*)"(?: within "([^"]*)")?$/ do |field, selector|
+When /^(?:|I )choose "(.*)"(?: within "([^"]*)")?$/ do |field, selector|
```

Comment:
This works because we never use the optional "within" clause, a greedy match would have eaten up any second parameter.

Reply:
Do you think it would be better to remove the unused clause or to set up a step definition like `When I opt to transfer my works/collections to the orphan account without my pseud attached`?

Reply:
Let's remove the unused clause for now.

Whenever we update to Cucumber 3.x, we can use [Cucumber expressions](https://cucumber.io/docs/cucumber/cucumber-expressions/), which allows this exact case:

https://github.com/cucumber/cucumber-expressions-ruby/blob/1c5871a622739e3424640515eab950f8cc3c1cec/spec/cucumber/cucumber_expressions/cucumber_expression_spec.rb#L42-L44